css/css.css

```
@@ -99,6 +108,12 @@ header {
   list-style: none;
   display: flex;
   justify-content: flex-end;
+  padding-left: 247px;
+  margin-top: 75px;
+}
+.basket-header-ikon {
```

Если начал добавлять пустую строку между селекторами, то нужно добавлять ее везде
```suggestion
}

.basket-header-ikon {
```